src/App.js

```diff
@@ -2,13 +2,58 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
-  render() {
-    return (
-      <main>
-        <p>React State Lab</p>
-      </main>
+constructor() {
+  super();
+  this.state = {
+    scoreNow: 0,
+    adding: 1,
+  };
+}
+
+handleClick = () => {
+  this.setState({
+    scoreNow: this.state.scoreNow + this.state.adding,
+  });
+};
+
+forUser = () => {
+  alert("You can't afford that!");
+};
+
+newScore = () => {
+  this.state.adding = this.state.adding + 1;
+  this.setState({ scoreNow: this.state.scoreNow - 10 });
+};
+
+playAnother = () => {
+  this.state.adding = 1;
+  this.setState({ scoreNow: 0 });
+};
```

Comment:
```suggestion
playAnother = () => {
  this.state.adding = 1;
  this.setState({ 
  scoreNow: 0,
  adding: 1
   });
};
```
You're modifying state directly on line 29. Again, **never ever ever modify state directly in React**! Always use `this.setState` if you need to update state.